src/contracts/GPv2Settlement.sol

```diff
@@ -300,6 +300,34 @@ contract GPv2Settlement {
         filledAmount[uidKey] = currentFilledAmount;
     }
 
+    /// @dev Allows settlment function to make arbitrary contract executions.
+    /// @param interaction contains address and calldata of the contract interaction.
+    function executeInteraction(GPv2Encoding.Interaction memory interaction)
+        internal
+    {
+        // To prevent possible attack on user funds, we explicitly disable
+        // interactions with AllowanceManager contract.
+        require(
+            interaction.target != address(allowanceManager),
+            "GPv2: forbidden interaction"
+        );
+        // solhint-disable avoid-low-level-calls
+        (bool success, bytes memory response) =
+            (interaction.target).call(interaction.callData);
+        // solhint-enable avoid-low-level-calls
+
+        // TODO - concatenate the following reponse "GPv2: Failed Interaction"
+        // This is the topic of https://github.com/gnosis/gp-v2-contracts/issues/240
+        if (!success) {
+            // Assembly used to revert with correctly encoded error message.
+            // solhint-disable no-inline-assembly
```

Comment:
Just FYI, disabling the next line should be enough (then you don't need the `// solhint-*` comment pair.
```suggestion
            // solhint-disable-next-line no-inline-assembly
```

The issue with the previous linter warning coming up when using `solhint-disable-next-line` is that the auto formatter decided to split the statement into two lines. This shouldn't be an issue here.